ecosystem/sep-0006.md

```diff
@@ -1045,6 +1045,7 @@ Name | Type | Description
 `amount_out_asset` | string | (optional) The asset delivered or to be delivered to the user. Must be present if the deposit/withdraw was made using quotes. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
 `amount_fee` | string | (optional) Amount of fee charged by anchor. Should be equals to `quote.fee.total` if a `quote_id` was used.
 `amount_fee_asset` | string | (optional) The asset in which fees are calculated in. Must be present if the deposit/withdraw was made using quotes. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format). Should be equals to `quote.fee.asset` if a `quote_id` was used.
+`quote_id` | string | (optional) The ID of the quote used to create this transaction. Should be present if a `quote_id` was included in the `POST /transactions` request.
```

Comment:
That makes sense to me. What do you think of the following wording?

```suggestion
`quote_id` | string | (optional) The ID of the quote used to create this transaction. Should be present if a `quote_id` was included in the `POST /transactions` request. Clients should be aware though that the `quote_id` may not be present if the Anchor implementation is prior to `v3.13.0`.
```

Reply:
Cc @JakeUrban, @leighmcculloch.

Reply:
The one thing I would say is that anchors don't have a way of specifying the version they're compatible with, and we haven't cited versions in the doc previously. I'm not against it, but it may be fine to just say that older implementations may not have the `quote_id` attribute.